fabric-commands-v0/src/main/java/net/fabricmc/fabric/mixin/command/MixinCommandManager.java

```diff
@@ -29,7 +29,7 @@
 import net.fabricmc.fabric.impl.command.CommandRegistryImpl;
 
 @Mixin(CommandManager.class)
-public class MixinCommandManagerIntegrated {
```

Comment:
I feel the Integrated part should stay since it distinctly mentions which logical server this applies to.

Reply:
The mixin was called that because CommandManager used to be called CommandManagerIntegrated so it felt right to change it here too.